superbit_lensing/metacalibration/ngmix_fit_superbit3.py

```diff
@@ -500,7 +500,7 @@ def mp_fit_one(source_id, jaclist, obslist, prior, logprint, pars=None):
 
     Tguess = 4*jaclist[0].get_scale()**2
     ntry = 4
-    psf_model = 'em3' # should come up with diagnostics for PSF quality
+    psf_model = 'gauss' # should come up with diagnostics for PSF quality
```

Comment:
I feel uncomfortable to keep pushing these kinds of changes to `master`...Ideally this would be something read in front a config or something. But certainly not committing a test setting

Reply:
Sure, we should definitely turn the `psf_model` into a config option in the future. However, I think that setting the default PSF model to `gauss` is preferable., especially in a test setting. It is faster and more robust to fitting failures than `em3`, and as we have discussed off-line, it's not clear that it offers any advantage for the SuperBIT PSF.